### PR TITLE
[Aqara] add new feature to K100

### PR DESCRIPTION
**Pull request by seojune79**
Check all that apply

# Type of Change

- [ ] WWST Certification Request
     - If this is your first time contributing code:
          - [ ] I have reviewed the README.md file
          - [ ] I have reviewed the CODE_OF_CONDUCT.md file
          - [ ] I have signed the CLA
     - [ ] I plan on entering a WWST Certification Request or have entered a request through the WWST Certification console at developer.smartthings.com
- [ ] Bug fix
- [x] New feature
- [ ] Refactor

# Checklist

- [ ] I have performed a self-review of my code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have verified my changes by testing with a device or have communicated a plan for testing
- [ ] I am adding new behavior, such as adding a sub-driver, and have added and run new unit tests to cover the new behavior

# Description of Change

# Summary of Completed Tests

### CODE REVIEW

**Comment by Kwang-Hui** [comment]
Hi, @lelandblue 
Please refer STSE-3341 for details. Thank you.

File: drivers/Aqara/aqara-lock/src/init.lua [hunk 1]
```diff
@@ -107,23 +110,33 @@ local function event_lock_handler(driver, device, evt_name, evt_value)
...
-    id = nil
-    label = nil
-  elseif evt_value == 0x80020000 then -- one-time password
-    id = "OTP_STANDALONE"
-    label = nil
-  else
-    id, label = credential_utils.find_userLabel(driver, device, evt_value)
-  end
-  device:emit_event(Lock.lock.unlocked({ data = { method = evt_name, codeId = id, codeName = label } }))
+local function event_unlock_indoor_handler(driver, device, evt_name, evt_value)
```
Comment (cjswedes) [inline drivers/Aqara/aqara-lock/src/init.lua:113]: Can you add tests for these handlers?

**Review by cjswedes — COMMENTED** [review]
LGTM, but would like to get confirmation of a manual test with the device before approval.

**Comment by Kwang-Hui** [comment]
> LGTM, but would like to get confirmation of a manual test with the device before approval.

Yes, SVT, QA test has done. please check the test result from `STSE-3341` ticket.

File: drivers/Aqara/aqara-lock/src/init.lua (hunk 1 above)
Comment (Kwang-Hui) [reply drivers/Aqara/aqara-lock/src/init.lua:113]: Is it ok to add a test case as a seperated pull-request?

**Review by cjswedes — APPROVED** [review]

**Review by dljsjr — APPROVED** [review]

**Pull request merged**